nhais/sequence/tests/component_test_ids_generators.py

```diff
@@ -0,0 +1,110 @@
+"""Module to test independence and coherence of message id, interchange id and transaction id generators."""
+
+import unittest
+
+import aioboto3
+from sequence.interchange_id import InterchangeIdGenerator
+from sequence.message_id import MessageIdGenerator
+from sequence.transaction_id import TransactionIdGenerator
+from utilities import test_utilities
+from utilities import config
+
+
+class ComponentTestIds(unittest.TestCase):
+
+    def setUp(self):
+        config.setup_config("MHS")
+        self.table_name = 'test_ids_generators_table'
+        self.key = 'transaction_id'
+        self.endpoint = config.get_config('DYNAMODB_ENDPOINT_URL', None)
+        self.region_name = 'eu-west-2'
```

Comment:
Are you required to supply region in tests?

Reply:
botocore.exceptions.NoRegionError: You must specify a region. <- exception if no region specified

Reply:
ok 👍